test/functional/services/remote/webdriver.ts

```diff
@@ -66,7 +66,8 @@ async function attemptToCreateCommand(
   log: ToolingLog,
   browserType: Browsers,
   lifecycle: Lifecycle,
-  logPollingMs: number
+  logPollingMs: number,
+  remoteSessionUrl: string
```

Comment:
Since it is passed via the env variable + only for Chrome/IE browser, what if we read it directly before session creation?
```
let session;
const remoteSessionUrl = process.env.REMOTE_SESSION_URL;
if (remoteSessionUrl) {
  session = await new Builder()
    .forBrowser(browserType)
    .withCapabilities(chromeCapabilities)
    .usingServer(remoteSessionUrl)
    .build();
} else {
  session = await new Builder()
    .forBrowser(browserType)
    .withCapabilities(chromeCapabilities)
    .setChromeService(new chrome.ServiceBuilder(chromeDriver.path).enableVerboseLogging())
    .build();
}
```

Reply:
Is that not what's happening already?